src/components/DocumentSelector/index.js

```diff
@@ -90,8 +94,9 @@ const withDefaultOptions = (types: Object, group: GroupType) =>
   (props: Props) =>
     <LocalisedDocumentSelector
       {...props}
-      defaultOptions={ () =>
-        Object.keys(types).map(value => {
+      defaultOptions={ () => {
+        const typeList = Object.keys(types)
```

Comment:
Not entirely sure why type keys were set on a constant, why this change?

Reply:
When updating the HTML to be accessible I found that screen readers were not announcing the list as a list when the type keys array was not separated despite using semantic HTML elements.

Reply:
Alternate solution for this (and the other instance of this in the Intro component) is to add a seemingly redundant ARIA role="list" to the <ul>, <ol> element.

Reply:
No, I mean why this:

```
Object.keys(types).map...
```

was changed to:

```
const typeList = Object.keys(types)
typeList.map...
```

?

Reply:
When chained, like it was before, the generated list does not get announced by VoiceOver as a list despite using `<ol>`, `<ul>`. It does get announced with the change. Or if it is kept as it was ARIA role="list" is still required in the HTML tags.

Reply:
I see... Very odd though. But I'm not too fussy about this, just wanted to know the reason. Thanks!

Reply:
I agree! Can't find a more definitive answer to why this is the case unfortunately.